pkg/kubeapiserver/authorizer/patch.go

```diff
@@ -1,8 +1,49 @@
 package authorizer
 
+import (
+	"sync"
+
+	"github.com/blang/semver/v4"
+)
+
 var skipSystemMastersAuthorizer = false
 
 // SkipSystemMastersAuthorizer disable implicitly added system/master authz, and turn it into another authz mode "SystemMasters", to be added via authorization-mode
 func SkipSystemMastersAuthorizer() {
 	skipSystemMastersAuthorizer = true
 }
+
+var (
+	minimumKubeletVersion *semver.Version
+	versionLock           sync.Mutex
+	versionSet            bool
+)
+
+// getMinimumKubeletVersion retrieves the set global minimum kubelet version in a safe way.
```

Comment:
oh, please also update the comment to include information about why these methods were added (to avoid the import cycle).

Reply:
done